packages/react-charting/src/components/GaugeChart/GaugeChart.styles.ts

```diff
@@ -0,0 +1,130 @@
+import { FontSizes, FontWeights, HighContrastSelector, HighContrastSelectorBlack } from '@fluentui/react';
+import { IGaugeChartStyleProps, IGaugeChartStyles } from './GaugeChart.types';
+
+export const getStyles = (props: IGaugeChartStyleProps): IGaugeChartStyles => {
+  const { theme, chartValueSize, width, height, className, lineColor, toDrawShape } = props;
+
+  return {
+    root: [theme.fonts.medium, 'ms-GaugeChart', { width }, className],
+
+    chart: {
+      display: 'block',
+      width,
```

Comment:
Lets have a minimum width and height for the gauge chart. Right now there is no min value for width and height in the demo site.

Reply:
Will do that in a separate PR for all non cartesian charts, as discussed